tests/integration/targets/setup_iis_configuration_file/tasks/main.yml

```
@@ -0,0 +1,41 @@
+- name: backup iis configuration
+  block:
+    - name: ensure IIS features are installed
+      ansible.windows.win_feature:
+        name: Web-Server
+        state: present
+        include_management_tools: True
+      register: feature_install
+
+    - name: reboot after feature install
+      ansible.windows.win_reboot:
+      when: feature_install.reboot_required
+
+    - name: stop IIS service
+      ansible.windows.win_service:
+        name: W3SVC
+        state: stopped
+
+    - name: get iis configuration file checksum
+      ansible.windows.win_stat:
+        path: '{{ iis_config_file_path }}\{{ iis_config_file_name }}'
+        checksum_algorithm: sha1
+      register: stat_result
+
+    - name: backup iis configuration file
+      ansible.windows.win_copy:
+        src: '{{ iis_config_file_path }}\{{ iis_config_file_name }}'
+        dest: '{{ iis_config_file_path }}\{{ iis_config_bck_file_name }}'
+        remote_src: yes
+      register: copy_result
+
+    - assert:
+        that:
+          - "stat_result.stat.checksum == copy_result.checksum"
```

We don't need the assertion and this is the task we want to add the handler notify on before after this runs we have an artifact to clear.

```suggestion
      notify:
      - restore iis configuration
```

---

Notify was added to Task: "backup iis configuration file" with `ansible.windows.win_copy`